### PR TITLE
fix the code to support mac

### PR DESCRIPTION
**Pull request by clevertension**
I have successfully build it on osx 10.9

### CODE REVIEW

File: libpcap_stuff.c [hunk 1]
```diff
@@ -16,8 +16,8 @@
 #include <string.h>
 #include <stdlib.h>
 #include <sys/ioctl.h>
-#include <pcap.h>
 #include <net/bpf.h>
+#include <pcap.h>
```
Comment (ghost) [inline libpcap_stuff.c:20]: ?

**Comment by clevertension** [comment]
the #include position switch can avoid "redefinition of ‘struct bpf_program’", please see this
http://forums.macrumors.com/showthread.php?t=624675

**Comment by clevertension** [comment]
Compared with #1 , This is the minimize code change to support OSX, thanks

**Comment by antirez** [comment]
Merged, it's years I don't look here but I plan to be more responsive, sorry. Compared to #1 this is more minimal and a non brainer, would be great to integrate things from #1 that are necessary for a better osx experience. For me it is not super important to support xcode at all.

File: libpcap_stuff.c (hunk 1 above)
Comment (EtherTyper) [inline libpcap_stuff.c:20]: @bl4z3r3d https://github.com/the-tcpdump-group/libpcap/blob/master/pcap/pcap.h

**Pull request merged**